ds_judgements_public_ui/sass/includes/govuk_overrides/_grid.scss

```diff
@@ -367,3 +398,103 @@
     padding-bottom: none;
   }
 }
+
+.govuk-grid-column-one-third__info_cards_home {
+  display: flex;
+  flex-direction: row;
+  gap: $space-4;
+
+  max-width: 24rem;
+  margin: $space-2;
+
+  background-color: colour-var("accent-background-light");
+  border-top: 4px solid $color-yellow;
+
+  &:hover {
+    text-decoration: none;
+    background-color: colour-var("accent-background");
+  }
+
+  @media (max-width: $grid-breakpoint-medium) {
+    min-width: 65%;
+    height: $space-6;
```

Comment:
I don't think we should use space values for dimension css as it should be adding spacing rather than setting dimension sizes.